CONTRIBUTING.md

```diff
@@ -1,23 +1,39 @@
 # Contributing to jsii
+Thanks for your interest in contributing to AWS JSII! :heart:
 
-We welcome contributions of any kind.
+This document describes how to set up a development environment and submit your
+contributions. Please read it carefully and let us know if it's not up-to date
+(or even better, submit a pull request with your corrections! :wink:).
 
-This section includes information for contributors.
+## Pre-requisites
+### Toolchain Requirements
+Due to the polyglot nature of `jsii`, the toolchain requirements are somewhat
+more complicated than for most projects. In order to locally develop `jsii`, you
+will need the following tools:
 
-## Development Environment
+- [Node 8.11.0] or later
+- An OpenJDK-8 distribution
+  + [Oracle's OpenJDK8]
+  + [Amazon Corretto 8]
+- [.NET Core 2.0] or later
+- [Python 3.6.5] or later
+- [Ruby 2.5.1] or later
 
-Since this repo produces artifacts for multiple programming languages using
-__jsii__, it relies on the following toolchains:
+[Node 8.11.0]: https://nodejs.org/download/release/v8.11.0/
+[Oracle's OpenJDK8]: http://openjdk.java.net/install/
+[Amazon Corretto 8]: https://aws.amazon.com/corretto/
+[.NET Core 2.0]: https://www.microsoft.com/net/download
+[Python 3.6.5]: https://www.python.org/downloads/release/python-365/
+[Ruby 2.5.1]: https://www.ruby-lang.org/en/news/2018/03/28/ruby-2-5-1-released/
 
- - [Node.js 8.11.0](https://nodejs.org/download/release/v8.11.0/)
- - [Java OpenJDK 8](http://openjdk.java.net/install/)
- - [.NET Core 2.0](https://www.microsoft.com/net/download)
- - [Python 3.6.5](https://www.python.org/downloads/release/python-365/)
- - [Ruby 2.5.1](https://www.ruby-lang.org/en/news/2018/03/28/ruby-2-5-1-released/)
+### Alterative: build in Docker
 
-Our CI/CD uses the "superchain" image from [aws-delivlib](https://github.com/awslabs/aws-delivlib). 
+We have built a Docker image with all the required tool, which we are using for
+our own CI/CD: the "superchain" image from [aws-delivlib].
 
-This image can also be used locally like this (note that initial build may take quite some time):
+[aws-delivlib]: https://github.com/awslabs/aws-delivlib
```

Comment:
Don't we want people to use the new superchain?

Reply:
Well it's not there yet 😅 but yeah, that'll be updated as soon as I can merge that :P